docker/install/ubuntu2004_install_llvm.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -euxo pipefail
+
+wget https://apt.llvm.org/llvm.sh
```

Comment:
Is there any way we can validate this script before executing it? Or can we revert back to the previous way of adding the apt repos? Executing arbitrary scripts from the internet is generally a bad thing 😸

Reply:
From their website (https://apt.llvm.org/) I tried this:
```
deb http://apt.llvm.org/focal/ llvm-toolchain-focal main
```
But the issue was that this one is missing LLVM-13. We are not using llvm-13 in cortexm image but it was installing before. 
cc @driazati

Reply:
I fixed this, thanks to @driazati pointers.